wandb: Starting wandb agent üïµÔ∏è
2021-11-02 08:48:49,730 - wandb.wandb_agent - INFO - Running runs: []
2021-11-02 08:48:50,338 - wandb.wandb_agent - INFO - Agent received command: run
2021-11-02 08:48:50,338 - wandb.wandb_agent - INFO - Agent starting run with config:
	ac_update_ratio: 2
	confidence_scale: 2.8167783096508527
	target_entropy: -1.7893158247400933
	use_entropy: True
2021-11-02 08:48:50,347 - wandb.wandb_agent - INFO - About to run command: python3 src/main.py --train --wandb
2021-11-02 08:48:55,358 - wandb.wandb_agent - INFO - Running runs: ['sxz75glf']
wandb: Currently logged in as: jjshoots (use `wandb login --relogin` to force relogin)
wandb: WARNING Ignored wandb.init() arg project when running a sweep
wandb: WARNING Ignored wandb.init() arg entity when running a sweep
-----------------------
Using Device cuda:0
-----------------------
wandb: wandb version 0.12.6 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.12.0
wandb: Syncing run 64906
wandb:  View project at https://wandb.ai/jjshoots/UA3SAC_gym
wandb:  View sweep at https://wandb.ai/jjshoots/UA3SAC_gym/sweeps/nktx8eqi
wandb:  View run at https://wandb.ai/jjshoots/UA3SAC_gym/runs/sxz75glf
wandb: Run data is saved locally in /home/taij/e2sac/wandb/run-20211102_084901-sxz75glf
wandb: Run `wandb offline` to turn off syncing.
slurmstepd: *** JOB 4907707 ON zeus400 CANCELLED AT 2021-11-02T10:23:35 ***
/home/taij/.conda/envs/robocar/lib/python3.8/multiprocessing/resource_tracker.py:216: UserWarning: resource_tracker: There appear to be 6 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
